src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -289,17 +476,79 @@
   word-break: break-word;
 }
 
-.pf-c-notification-drawer__group-toggle-count {
-  margin-right: var(--pf-c-notification-drawer__group-toggle-count--MarginRight);
-  margin-left: auto;
+// Task group toggle
+.pf-c-notification-drawer__task-group-toggle {
+  display: grid;
+  align-items: center;
+  width: 100%;
+  grid-template-columns: 1fr auto auto;
+  padding: var(--pf-c-notification-drawer__task-group-toggle--PaddingTop) var(--pf-c-notification-drawer__task-group-toggle--PaddingRight) var(--pf-c-notification-drawer__task-group-toggle--PaddingBottom) var(--pf-c-notification-drawer__task-group-toggle--PaddingLeft);
+  background-color: var(--pf-c-notification-drawer__task-group-toggle--BackgroundColor);
+  border: solid var(--pf-c-notification-drawer__task-group-toggle--BorderColor);
+  border-width: 0 0 var(--pf-c-notification-drawer__task-group-toggle--BorderBottomWidth) 0;
```

Comment:
Typically this and the line above are just written as `border-bottom: var(--pf-c-notification-drawer__task-group-toggle--BorderBottomWidth) solid var(--pf-c-notification-drawer__task-group-toggle--BorderBottomColor);`

Reply:
@mcoker The border-width is turning off the other borders that would be coming from the button styling. This is taken from how the `.pf-c-notification-drawer__group-toggle` is done. Do you have a better way that also handles the other borders?

Reply:
Gotcha, wasn't thinking about the button's default borders.